app/controllers/tags_controller.rb

```diff
@@ -0,0 +1,7 @@
+class TagsController < ApplicationController
+  def show
+    @tag = Tag.find(params[:id])
+    @communities = Community.joins(:tag_references).where(tag_references: { tag_id: @tag.id })
```

Comment:
Skaties, šis kods tev vispar nav vajadzigs, tāpēc kā iekš tag ir jau logika lai radit visus attiecigos posts un communities.
Tu talak vari skatā izmantot @tag instances maingo un izsaukt @tag.posts vai @tag.communitites